images_metadata.yaml

```diff
@@ -0,0 +1,165 @@
+---
+fig_A.1:
+  caption: "Entity terms"
+  source: ISO 14812, Figure A.1
```

Comment:
Can we normalize the source in the way it is done in Geolexica, e.g. the clause "Figure A.1" and the document reference "ISO 14812:2022" are separated? Thanks.